src/composables/websocket.ts

```diff
@@ -0,0 +1,155 @@
+import { ref } from '@vue/composition-api';
+import camelcaseKeysDeep from 'camelcase-keys-deep';
+import { IDefaultComposableOptions, INetwork, WebSocketChannelTypes } from '../types';
+import { createNetworkWatcher } from './composablesHelpers';
+import { useGetter } from './vuex';
+
+const { onNetworkChange } = createNetworkWatcher();
+
+const wsClient = ref<WebSocket>();
```

Comment:
After spending some time on reviewing the code I feel that this doesn't need to be a Vue ref. Instead just regular `let`.